skinlist.lua

```diff
@@ -5,7 +5,7 @@ local dbgprint = false and print or function() end
 local function process_skin_texture(path, filename)
 	-- See "textures/readme.txt" for allowed formats
 
-	local prefix, sep, identifier, extension = filename:match("^(%a+)([_.])([%w_]+)%.(%a+)$")
+	local prefix, sep, identifier, extension = filename:match("^(%a+)([_.])([%w_.]+)%.(%a+)$")
```

Comment:
This line fixes #104.

Reply:
#101 The PR says it cannot find a skin IF the player has an underscore in it, the skin name does not have an underscore in it.

```
2024-06-10 14:33:37: WARNING[Server]: skinsdb: cannot find matching skin 'player_playername_with_underscore_2' for player 'playername_with_underscore'.
```

We already renamed all the skins, because we had a dot separator setting already. Please note, while YL tries to stick to the naming scheme `player_playername_number.png`, other servers may happily name theirs `player_playername_with_underscores_skinname_with_underscores.png`

Playername examples were along the lines of

```
Playername_II
_PLAYERNAME_
```

in modstorage it looked like this:

```
player_Playername_2
player_PLAYERNAME_0
```

NOK

#104 Private skins show up in the right list again OK